sql/moz-fx-data-shared-prod/telemetry_derived/suggest_clients_daily_v1/query.sql

```diff
@@ -118,21 +118,20 @@ clients AS (
   SELECT
     submission_date,
     client_id,
-    CAST(
+    COALESCE(CAST(
       user_pref_browser_urlbar_quicksuggest_data_collection_enabled = "true" AS bool
-    ) AS user_pref_data_collection_enabled,
```

Comment:
You don't need the cast here (`x = y` is already a boolean), you can do

```
COALESCE(
  user_pref_browser_urlbar_quicksuggest_data_collection_enabled = "true",
  FALSE
) AS  user_pref_data_collection_enabled
```

(same for the 2 below)